wdio.conf.js

```diff
@@ -1,12 +1,6 @@
-const path = require('path');
 // eslint-disable-next-line import/no-extraneous-dependencies
-const defaultWdioConfig = require('@cerner/terra-functional-testing/lib/config/wdio.conf');
+const { config } = require('@cerner/terra-functional-testing');
 
-const wdioConfig = defaultWdioConfig.config;
+config.specs = ['./packages/**/wdio/**/*-spec.js'];
```

Comment:
Do we no longer need "tests" in the path?  It appears to be there in the wdioConfig.specs path on line #9

Reply:
It would still work because the packages/** would cover any directory structure in between.  It just would not limit to the specific path that was in place before.